### PR TITLE
Review and tweak Landsat code

### PR DESCRIPTION
**Pull request by alexgleith**
Minor changes all over the place.

I removed the SQS class, because it was only being used in Landsat for one little thing. I used code that's existing elsewhere.

### CODE REVIEW

File: dags/landsat_scenes_sync/landsat_scenes_fill_gaps.py [hunk 1]
```diff
@@ -226,9 +215,8 @@ def fill_the_gap(landsat: str, scenes_limit: int) -> None:
         tags=["Landsat_scenes", "fill the gap"],
         catchup=False,
 ) as dag:
-    START = DummyOperator(task_id="start-tasks")
-
     processes = []
+    # Does Landsat 5 needs to have a capital L?
```
Comment (carioca-au) [inline dags/landsat_scenes_sync/landsat_scenes_fill_gaps.py]: no I have to change that... good catch

File: dags/landsat_scenes_sync/landsat_scenes_gap_report.py [hunk 2]
```diff
@@ -116,17 +115,15 @@ def build_path(file_row):
         for row in read_big_csv_files_from_gzip(file_path)
         if (
             # Filter to skip all LANDSAT_4
-            row.get("Satellite")
-            and row["Satellite"] != "LANDSAT_4"
+            row.get("Satellite") is not None and row["Satellite"] != "LANDSAT_4"
             # Filter to get just day
             and (
-                row.get("Day/Night Indicator")
+                row.get("Day/Night Indicator") is not None
```
Comment (carioca-au) [inline dags/landsat_scenes_sync/landsat_scenes_gap_report.py:121]: Why did you add **is not None**, here?

File: dags/landsat_scenes_sync/landsat_scenes_gap_report.py [hunk 3]
```diff
@@ -116,17 +115,15 @@ def build_path(file_row):
...
                 and row["Day/Night Indicator"].upper() == "DAY"
             )
             # Filter to get just from Africa
             and (
-                row.get("WRS Path")
-                and row.get("WRS Row")
+                row.get("WRS Path") is not None and row.get("WRS Row") is not None
```
Comment (carioca-au) [inline dags/landsat_scenes_sync/landsat_scenes_gap_report.py:126]: Same as above

**Review by carioca-au — APPROVED** [review]

File: dags/landsat_scenes_sync/landsat_scenes_gap_report.py (hunk 2 above)
Comment (alexgleith) [reply dags/landsat_scenes_sync/landsat_scenes_gap_report.py:121]: https://stackoverflow.com/questions/3965104/not-none-test-in-python

File: dags/landsat_scenes_sync/landsat_scenes_fill_gaps.py [hunk 4]
```diff
@@ -188,30 +176,31 @@ def fill_the_gap(landsat: str, scenes_limit: int) -> None:
...
             ]
 
-            limit = scenes_limit if scenes_limit else len(missing_scene_paths)
-
-            logging.info(f"missing_scene_paths {missing_scene_paths[0:10]}")
+            logging.info(f"Number of scenes found {len(missing_scene_paths)}")
+            logging.info(f"Example scenes: {missing_scene_paths[0:10]}")
 
-            logging.info(f"Number of scenes found {len(missing_scene_paths)} limited in - {limit}")
+            if scenes_limit is not None:
```
Comment (alexgleith) [inline dags/landsat_scenes_sync/landsat_scenes_fill_gaps.py:189]: Will this be `0` @carioca-au?

Seems to me it'll either be an `int` or `None`.
